src/main/java/techcourse/myblog/domain/ArticleRepository.java

```diff
@@ -4,12 +4,31 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Repository
 public class ArticleRepository {
-    private List<Article> articles = new ArrayList<>();
+    private final List<Article> articles = new ArrayList<>();
 
-    public List<Article> findAll() {
-        return articles;
+    public List<ArticleDto> findAll() {
+        return articles.stream()
```

Comment:
DTO를 구현한 아이디어가 매우 좋았습니다. 👍 하지만 `ArticleDto`가 아닌 `Article`를 반환하도록 수정해야 다음 미션을 원활히 진행할 수 있을 거예요. REPOSITORY는 `Article`에만 의존하도록 변경 부탁드려요.

```suggestion
        return new ArrayList<>(articles);
```